default.nix

```diff
@@ -71,7 +71,8 @@ let
     filter = localLib.isPlutus;
   };
   customOverlays = optional forceError errorOverlay;
-  packages = self: ({
+  purescriptNixpkgs = import (localLib.iohkNix.fetchNixpkgs ./plutus-playground/plutus-playground-client/nixpkgs-src.json) {};
```

Comment:
Still suspicious of this.

Reply:
@krisajenkins is this necessary now?

Reply:
I think we should try without. I put it in because there was something we needed at the time that was broken in the main Plutus nixpkgs. But it's worth checking if that's still the case.

Obviously, ideally, we'd only need one nixpkgs.

Reply:
ok well I'm just fixing the styling then I'll try removing it

Reply:
ah, pinned pkgs uses purescript 0.12 but we need to use 0.11.7 (for now)

Reply:
Can we override it instead of getting a whole separate nixpkgs?

Reply:
I'm just a bit worried that this is going to break in subtle ways that we don't predict.

Reply:
*sigh*

okay